dev-support/docker/Dockerfile

```diff
@@ -248,4 +248,4 @@ RUN curl -L -o hugo.deb https://github.com/gohugoio/hugo/releases/download/v0.30
 COPY hadoop_env_checks.sh /root/hadoop_env_checks.sh
 RUN chmod 755 /root/hadoop_env_checks.sh
 # hadolint ignore=SC2016
-RUN echo '${HOME}/hadoop_env_checks.sh' >> /root/.bashrc
+RUN echo '${HOME}/hadoop_env_checks.sh' >> /root/.bashrc
```

Comment:
You shouldn't need to modify this file at all.

Reply:
@GauthamBanasandra I will rollback the file changes.